tests/test_content_loader.py

```diff
@@ -934,7 +980,7 @@ def test_get_non_multiquestion_question_as_section(self):
 
         question_section = section.get_question_as_section('q1-slug')
         assert question_section.slug == "q1-slug"
-        assert question_section.description == "Some description"
+        assert question_section.description == ""
```

Comment:
Can't quite figure out if this change in behaviour will break any existing things...